fuzzers/032-cmt-pll/generate.py

```diff
@@ -10,7 +10,15 @@ def bitfilter(frame, word):
     if frame < 28:
         return False
 
-    if frame == 25 and word == 3121:
+    # Those bits are in conflict with fuzzer 034.
+    if (frame, word) in (
+        (28, 10),
```

Comment:
I'm not a huge fan of hard coding conflicts.  Any way we can vary the input stimulus to avoid this kind of filter list?  034 clearly toggles these bits, can we modify 032 to do the same?

Reply:
That would probably require routing manually through some pips. I can take a look at it.

Reply:
Thanks, I think this is fuzzer is better for those changes!